.gitignore

```diff
@@ -2,3 +2,4 @@
 Makefile
 *.pyc
 __pycache__/
+/.idea/
```

Comment:
Not entirely sure what thiis is. However I suspect this is better handled locally your end by either adding it into your global .gitignore or in the repository under .git/info/exclude